docs/7.x/pack.md

```diff
@@ -123,6 +123,7 @@ The command below assumes that `build.sh` is located in the same working directo
 docker run \
        -v /tmp/tele-cache:/mnt/tele-cache \
        -v /var/run/docker.sock:/var/run/docker.sock \
+       -v $(which docker):/usr/bin/docker \
```

Comment:
Hmm, I think that mounting docker binary inside the container may be asking for trouble, potentially issues with shared libraries and so on so I'm not sure I would recommend this in the docs... why not just install docker client in the container?

Reply:
Yeah, I realized this when CI hit the shared libraries issue.  Elsewhere (`assets/robotest/run.sh` where I was testing) mounting the binary saved a docker file and the extra build targets.  I'm reworking this approach presently exploring @a-palchikov's advice here: https://github.com/gravitational/gravity/pull/2150#discussion_r494862308

Reply:
Robotest image build fixed in 0e93513 . Docs fixed in 66ea6db.